Automattic/wp-calypso
# JCloud: apply some styles on Backup Status views for backup success & schedule

#### Changes proposed in this Pull Request

Apply some styles on Backup Status views for backup success & schedule. Add status icons for successful backup and for backup scheduled.

Before:
![image](https://user-images.githubusercontent.com/9832440/82075344-020f1f00-96d4-11ea-9a0a-5530a5cc3b82.png)

Now: 
![image](https://user-images.githubusercontent.com/9832440/82074790-32a28900-96d3-11ea-9e02-b4b98e48cad4.png)

Before:
![image](https://user-images.githubusercontent.com/9832440/82075551-4e5a5f00-96d4-11ea-9c40-c0c6f0f7111b.png)

Now:
![image](https://user-images.githubusercontent.com/9832440/82075296-ed328b80-96d3-11ea-9dac-ec5d0e16a675.png)

Before:
![image](https://user-images.githubusercontent.com/9832440/82075623-6cc05a80-96d4-11ea-99d8-e480e32aba98.png)

Now:
![image](https://user-images.githubusercontent.com/9832440/82074835-44842c00-96d3-11ea-8a1a-ce6c064a8f20.png)


**This PR depend on:** https://github.com/Automattic/wp-calypso/pull/42259

#### Testing instructions

- Apply the changes
- Check if it matches with the designs.
